### DISCUSSION TITLE
Complete process of createing a skeleton and program in cokreat

### DISCUSSION THREAD

**Discussion by shubhambansaltarento** [Sunbird-coKreat/Community#64] [Q&A]
Can any one help me in knowing a complete process of how to create Project Skeleton from Consumption portal and access from sourcing portal and create program's with it locally or on dev/local server ?

**Comment by mmanimegalai** [comment]
@shubhambansaltarento 
Please refer https://cokreat.sunbird.org/learn/product-and-developer-guide/asset-sourcing/creation-of-sourcing-projects to understand how to create a project/program

@arun1205 Please provide any other relevant details that will help
